src/main/java/marites/Marites.java

```diff
@@ -0,0 +1,312 @@
+import marites.task.Event;
+import marites.task.Task;
+import marites.task.Todo;
+
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+import java.io.File;
+
+import static java.lang.Integer.parseInt;
+
+public class Marites {
+
+    /* STRING CONSTANTS */
+
+    // Logo found in https://emojicombos.com/kaomoji
+    private static final String INTRO_MESSAGE = "(งツ)ว\n" +
+            "Hi, I'm Marites! I've heard so many things about you!\n" +
+            "I have a lot of stories to share, but first, how can I help you?\n";
+    private static final String EXIT_MESSAGE = "See you next time!";
+    private static final String UNKNOWN_COMMAND_MESSAGE = "I didn't understand that.\n";
+
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_ADD_TODO = "todo";
+    private static final String COMMAND_ADD_DEADLINE = "deadline";
+    private static final String COMMAND_ADD_EVENT = "event";
+    private static final String COMMAND_ADD_DEADLINE_TAG = "--by";
+    private static final String COMMAND_ADD_EVENT_TAG = "--at";
+    private static final String COMMAND_DELETE = "delete";
+
+    private static final String LIST_TASK_ITEM_FORMAT_STRING = "%d. %s\n";
+    private static final String ADD_TASK_FORMAT_STRING =
+           "Alright, task added:%n  %s%nYour list currently has %d tasks.\n";
+    private static final String ADD_TASK_MISSING_PARAMETER_MESSAGE =
+           "You're missing a parameter: %s\n";
+    private static final String ADD_TASK_MISSING_DESCRIPTION_MESSAGE =
+            "Please add a description to your task.\n";
+    private static final String ADD_TASK_UNKNOWN_TASK_MESSAGE =
+            "I don't know this task type: '%s'\n";
+    private static final String MARK_DONE_MESSAGE = "Good job on getting this done!";
+    private static final String MARK_UNDONE_MESSAGE = "Okay, I've marked this as not yet done:";
+    private static final String DELETE_TASK_MESSAGE =
+            "Alright, task deleted:%n  %s%nYour list currently has %d tasks.\n";
+    private static final String INVALID_TASK_NUMBER_MESSAGE =
+            "I either didn't get a task number, or couldn't understand it: '%s'\n";
+    private static final String SET_TASK_STATUS_FORMAT_STRING = "%s%n    %s\n";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+    public static final String SAVE_FILENAME = "tasklist.ser";
+    public static final String READ_TASK_LIST_ERROR_MESSAGE = "WARNING: Error when reading saved task list; initializing with empty list";
+    public static final String SAVE_TASK_LIST_ERROR_MESSAGE = "WARNING: Error occurred while saving task list";
```

Comment:
You might want to extract these constants out and create a dedicated helper class.